package/yast2-schema.spec

```diff
@@ -35,6 +35,12 @@ BuildRequires:	yast2-online-update-configuration
 
 # yast2-core omited from some reason in buildservice, adding explicitly
 BuildRequires:  yast2-core
+
+# openSUSE does not contain the registration module
+%if %suse_version == 1315
```

Comment:
I see. I found [this one](https://github.com/yast/yast-packager/blob/master/package/yast2-packager.spec#L79).